### PR TITLE
[PL-49092] Quartz Exp is valid, while Unix Exp is invalid for the Cron

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-49092](https://harness.atlassian.net/browse/PL-49092)

More info: https://harness.atlassian.net/browse/PL-48951

Adding a note to the Cron Expression section

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-49092]: https://harness.atlassian.net/browse/PL-49092?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/platform/authentication/single-sign-on-sso-with-ldap.md [hunk 1]
```diff
@@ -245,6 +245,25 @@ To configure a synchronization schedule, in **Enter a custom cron expression**,
...
+| | | | |  |  
+| | | | |  \-------- YEAR (2014)  
+| | | | \----------- DAY_OF_WEEK (NOT_SPECIFIED)  
+| | | \------------- MONTH (JULY)  
+| | \--------------- DAY_OF_MONTH (4th)  
+| \----------------- HOUR (0- MIDNIGHT LOCAL TIME)  
+\------------------- MINUTE (0)
+```
+
+For more information, please refer [Cron Expressions](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm) documentation.
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-sso-with-ldap.md:264]:
```suggestion
For more information, go to [Cron Expressions](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm) in the Oracle documentation.
```

File: docs/platform/authentication/single-sign-on-sso-with-ldap.md [hunk 2]
```diff
@@ -245,6 +245,25 @@ To configure a synchronization schedule, in **Enter a custom cron expression**,
 
 The page includes a tabular breakdown of the cron expression for ease of understanding and verification. The **Cron expression** field shows you whether or not the cron expression is valid.
 
+:::note
+The Cron(Quartz) Expression field will accept string consisting of six or seven subexpressions (fields) that describe individual details of the schedule. Unix Cron Expression consisting of five subexpressions is not supported.
+
+A sample of the Quartz expression is below:
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-sso-with-ldap.md:251]:
```suggestion
Here's a sample Quartz expression:
```

File: docs/platform/authentication/single-sign-on-sso-with-ldap.md [hunk 3]
```diff
@@ -245,6 +245,25 @@ To configure a synchronization schedule, in **Enter a custom cron expression**,
 
 The page includes a tabular breakdown of the cron expression for ease of understanding and verification. The **Cron expression** field shows you whether or not the cron expression is valid.
 
+:::note
+The Cron(Quartz) Expression field will accept string consisting of six or seven subexpressions (fields) that describe individual details of the schedule. Unix Cron Expression consisting of five subexpressions is not supported.
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-sso-with-ldap.md:249]:
```suggestion
The **Cron Expression** field accepts a Quartz CronTrigger string consisting of six or seven subexpressions (fields) that describe individual details of the schedule. Unix cron expressions consisting of five subexpressions are not supported.
```

**Review by brian-f-harness — COMMENTED** [review]
Some suggested updates

File: docs/platform/authentication/single-sign-on-sso-with-ldap.md [hunk 4]
```diff
@@ -245,6 +245,25 @@ To configure a synchronization schedule, in **Enter a custom cron expression**,
 
 The page includes a tabular breakdown of the cron expression for ease of understanding and verification. The **Cron expression** field shows you whether or not the cron expression is valid.
 
+:::note
+The **Cron Expression** field accepts a Quartz CronTrigger string consisting of six or seven subexpressions (fields) that describe individual details of the schedule. Unix cron expressions consisting of five subexpressions are not supported.
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-sso-with-ldap.md:249]: nit

Sync The field name capitalization for **Cron expression**. On line 246, it's lowercase. Here it's uppercase. Verifying the UI and match the capitalization.

**Review by brian-f-harness — APPROVED** [review]

File: docs/platform/authentication/single-sign-on-sso-with-ldap.md (hunk 4 above)
Comment (saifeemustafaqharness) [reply docs/platform/authentication/single-sign-on-sso-with-ldap.md:249]: Thankyou for the edit Brian, appreciate the changes suggestion.

**Pull request merged**